src/service.rs

```diff
@@ -15,7 +16,7 @@ pub struct HcHttpGatewayService {
 }
 
 /// Shared application state
-#[derive(Debug, Clone)]
+#[derive(Clone)]
```

Comment:
Do we need to implement debug for `AppInfo`?

Reply:
If so, we might want to wrap the new field in a newtype and implement debug for that for now